### DISCUSSION TITLE
i really need help

### DISCUSSION THREAD

**Discussion by sasukekun666** [Rem0o/FanControl.Releases#889] [Q&A]
hi guys I'm very new to this. i literally just updated my Radeon rx570 drivers and all of a sudden, Speedfan stopped showing me my GPU fan speed and didn't show any of the stats on there. so then I've downloaded FanControl and its recognising the GPU is there but it doesn't show up on the fan control tab. it just recognises the temp of the GPU and displays it in my sidebar. if anyone can help, it;d be greatly appreciated.  
![Picture1](https://user-images.githubusercontent.com/108027015/175120838-b1b53b8c-5dde-4dd4-8cce-6240d47f753b.png)

**Comment by AtA3301** [comment]
Try a clean install of the drivers - use DDU to remove all trace of the GPU and start again with the latest non-beta release.

**Reply by lwk123qq** [reply]
https://www.bilibili.com/video/BV14a411L7Vi?spm_id_from=333.337.search-card.all.click&vd_source=df7407586d660cd45dc8c353960cec30

你可以参考这个教程
